cf-agent/package_module.c

```diff
@@ -41,13 +42,16 @@
 static bool UpdateSinglePackageModuleCache(EvalContext *ctx,
                                     const PackageModuleWrapper *module_wrapper,
                                     UpdateType type, bool force_update);
-static char *GetPackageModuleRealPath(const char *package_manager_name);
+static void GetPackageModuleExecInfo(const PackageModuleBody *package_module, char **exec_path,
+                                     char **script_path, char **script_exec_opts);
 static int NegotiateSupportedAPIVersion(PackageModuleWrapper *wrapper);
 
 
 void DeletePackageModuleWrapper(PackageModuleWrapper *wrapper)
 {
     free(wrapper->path);
+    free(wrapper->script_path);
```

Comment:
These `Delete` functions usually work on `NULL` pointers, (doing nothing).

I would add that `if` here, or an `assert(wrapper != NULL)`.